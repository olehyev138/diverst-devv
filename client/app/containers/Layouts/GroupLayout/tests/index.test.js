/**
 *
 * Tests for GroupLayout
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import { shallow } from 'enzyme';
import MockGroupLayout from './mock';

describe('<GroupLayout />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    const wrapper = shallow(<MockGroupLayout classes={{}} />);

    expect(spy).not.toHaveBeenCalled();
  });
});
