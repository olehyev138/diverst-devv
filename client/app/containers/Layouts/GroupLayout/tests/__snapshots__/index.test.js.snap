// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GroupLayout /> Should render and match the snapshot 1`] = `
<GroupLayout
  classes={
    Object {
      "content": "GroupLayout-content-2",
      "toolbar": "GroupLayout-toolbar-1",
    }
  }
  component={[MockFunction]}
/>
`;
